iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```
@@ -1070,6 +1070,13 @@ static LogicalResult convertPadTensorOp(OpBuilder &b,
       *padTensorOp.region().getOps<linalg::YieldOp>().begin();
   Value paddingValue = yeildOp.values()[0];
 
+  auto constOp = paddingValue.getDefiningOp<ConstantOp>();
+  assert(constOp &&
```

Instead of asssert, it is better to return a failure.